web/.stylelintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "stylelint-config-standard-scss",
+    "rules": {
+        "color-function-notation": "legacy",
+        "alpha-value-notation": "number",
+        "selector-class-pattern": [
+			"^([a-z][a-z0-9]*)(-[a-z0-9]+)*(__([a-z][a-z0-9]*)(-[a-z0-9]+)*)*$",
```

Comment:
This is done to allow some default classes of `Material Design`.
It has the format: `classname-word2__after-separator`